.github/CODEOWNERS

```diff
@@ -9,3 +9,4 @@ src/locales/fr-CH @p1d1d1
 src/locales/it @p1d1d1
 src/locales/it-CH @p1d1d1
 src/locales/ro @mneagul
+src/locales/pt-BR @uba
```

Comment:
I've sent you an invite for this repository. Otherwise the codeowners file won't recognize you.